eng/Version.Details.xml

```diff
@@ -75,22 +75,22 @@
       <Uri>https://github.com/dotnet/xliff-tasks</Uri>
       <Sha>a52f3d7fb58470749ee4035fbbcb7e63c78b0459</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="5.0.0-rc.1.20451.17" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="5.0.0" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
```

Comment:
This is the wrong package, should be `Microsoft.AspNetCore.App.Runtime.win-x64` to avoid missing servicing updates.

Reply:
Updated.